docs/source/app_developers_guide/overview.rst

```diff
@@ -0,0 +1,36 @@
+***************************
+Transaction Family Overview
+***************************
+
+Sawtooth separates the application level from the core system level with
+transaction families, which allows application developers to write in the
+languages of their choice. Each application defines the custom transaction
+transaction families for its unique requirements.
+
+A transaction family includes these components:
+
+* A transaction processor to define the business logic for your application
+* A data model to record and store data
+* A client to handle the client logic for your application
+
+The transaction processor is responsible for registering with the validator,
+providing a transaction handler with an apply method and a transaction handler
```

Comment:
This paragraph could be revised, not quite sure what "providing a transaction handler with an apply method and a transaction handler class..." means